HouseholdManager/Views/Room/Details.cshtml

```diff
@@ -1,38 +1,46 @@
-﻿@model HouseholdManager.Models.Room
+﻿@model HouseholdManager.Models.ViewModels.RoomDetailViewModel
 @using HouseholdManager.Data.API
 
 @{
-    ViewData["Title"] = "Room Details";
+    ViewData["Title"] =  @Model.RoomName + " Details";
     var req = new IconRequestor();
-    string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,
-                                                                 (IEnumerable<Icon>)ViewBag.Icons);
+    //Below is no longer needed with the ViewModel in place
+    //string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,
+    //                                                             (IEnumerable<Icon>)ViewBag.Icons);
 }
 
-<h1>Details</h1>
-@await Html.PartialAsync("_CreationButtonsPartial")
+<h1>@Model.RoomIcon @Model.RoomName Details</h1>
```

Comment:
The buttons on this page seem to be from an earlier version before the button icon/consistency pull, not totally sure why though - usually version control prevents that. Anyway, that's what the rest of these suggestions are about.

```suggestion
<h1><span class="icon-font" title=@iconMouseOverText>@Model.RoomIcon</span> @Model.RoomName</h1>
@await Html.PartialAsync("_CreationButtonsPartial")
```

Restores the newer buttons, changes the icon in the h1 to use the same icon font as the rest of the app